src/generic/grid.cpp

```diff
@@ -4573,6 +4722,66 @@ void wxGrid::EnableDragColMove( bool enable )
     // order, whereas now you can always call ResetColPos() manually if needed
 }
 
+bool wxGrid::FreezeTo(unsigned row, unsigned col)
+{
+    if( row >= m_numRows || col >= m_numCols )
+        return false;
+    
+    // freeze
+    if( row > m_numFrozenRows || col > m_numFrozenCols )
+    {
+        // check that it fits in client size
```

Comment:
I haven't thought about what problems it may lead to exactly, I just saw that the other apps with the freeze feature don't allow it, so I did so as well.

Reply:
Actually while testing google spreadsheet I remembered why I added this condition.
Without it you can scroll a lot in one direction and then apply the freeze, which would work, and leave you unable to scroll anymore on that axis.